unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/CharacterPreviewController.cs

```diff
@@ -54,6 +54,9 @@ private void Awake()
             {CameraFocus.FaceSnapshot, faceSnapshotTemplate},
             {CameraFocus.BodySnapshot, bodySnapshotTemplate},
         };
+        var cullingSettings = DCL.Environment.i.platform.cullingController.GetSettingsCopy();
+        cullingSettings.ignoredLayersMask |= 1 << CHARACTER_PREVIEW_LAYER;
+        DCL.Environment.i.platform.cullingController.SetSettings(cullingSettings);
```

Comment:
Wouldn't it be better to have this in the CullingController initialization?

Approving anyway!

Reply:
I thought about that in a first approach. My main concern was that unity force you to ask for the layer value using a string, this means that if in the future we change the name the most probable scenario is that we forget to change it on the CullingController and we end up we this issue again :P
I couldn't use the constant in this class cause to do that cause that will require to reference Avatar Editor asmdef in the Culling Controller asmdef.
So the other viable option was to use the Config script for that, but I found it kind of an overkill for something so specific